packages/frontend/src/components/ControlledAutocomplete/AddNewOptionModal.tsx

```diff
@@ -49,6 +54,25 @@ export function useCreateNewOption(setValue: (newValue: string) => void) {
             newValue = data?.createTable.id
             break
           }
+          case 'tiles-createTileRow-columnId': {
+            const tableId = parameters?.tableId
+            if (!tableId) {
+              return
+            }
```

Comment:
can do this to fix type issues

```suggestion
            const tableId = parameters?.tableId as string // confirm plus chop
            if (!tableId) {
              return
            }
```

or alternatively this but tbh this is no better than a hard cast because we're returning silently.

```suggestion
            const tableId = parameters?.tableId
            if (typeof tableId !== 'string' || !tableId) {
              return
            }
```

Reply:
fixed